### PR TITLE
Add Travis CI builds

### PR DESCRIPTION
**Pull request by cajun-rat**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+dist: trusty
+sudo: false
+language: cpp
+env:
+  - DOCKERFILE=Dockerfile
+  - DOCKERFILE=Dockerfile.ubuntu-xenial
+services:
+  - docker
+script:
+  - docker build -t advancedtelematic/sota-tools -f ${DOCKERFILE} .
```
Comment (stevana) [inline .travis.yml]: Would be nice if it would do a docker push on success also (https://docs.travis-ci.com/user/docker/).

Comment (cajun-rat) [inline .travis.yml]: If it would be useful, sure.  Do you know the incantation that is required?

Comment (stevana) [inline .travis.yml]: If you follow the above link there's a section called "Pushing a Docker Image to a Registry", we probably want something like what's described under "Branch Based Registry Pushes".

Comment (cajun-rat) [inline .travis.yml]: I'll have to talk to @alaa and @alexhumphreys about how to set this up.  Those instructions require a docker username/password, and I'd be much happier if travis ci didn't have permission to push arbitrary docker images at us.

Comment (alexhumphreys) [inline .travis.yml]: From the looks of it there's no way around giving them a username and password. But we could create a new `ats-travis` user dockerhub user that only has access to one, or however many, advancedtelematic dockerhub repos it needs.

Comment (cajun-rat) [inline .travis.yml]: That sounds like a good solution.  Do you want to attack this tomorrow sometime?

Comment (alexhumphreys) [inline .travis.yml]: There is now a user called atstravis on dockerhub, and a travis team in the advancedtelematic organisation. Add that team as a colalborator on any of the repositories you want them to have access on. I'm away from tomorrow till next wednesday, but @alaa should be able to help you if anything comes up.

Comment (cajun-rat) [inline .travis.yml]: OK, I think this now works

**Pull request merged**